lib/plugins/physics.js

```diff
@@ -128,7 +134,7 @@ function inject(bot) {
 
     // limit speed
     var currentMaxGroundSpeed;
-    var underBlock=bot.blockAt(pos.offset(0,-1,0));
+    var underBlock=bot.blockAt(new Vec3(pos.x,Math.ceil(pos.y-1.002),pos.z));
```

Comment:
Sounds good. Stuff like this should be documented (at least by using constants with talking names), or the code quickly gets difficult to understand.